ampd/Cargo.toml

```diff
@@ -35,7 +35,6 @@ move-core-types = { git = "https://github.com/mystenlabs/sui", tag = "mainnet-v1
 multisig = { workspace = true }
 num-traits = { workspace = true }
 prost = "0.11.9"
-prost-types = "0.11.9"
```

Comment:
Why remove this?

Reply:
There are a ton of conflicts before merge. I talked with Chris. I have all other files on local and will push when we decide on what to do with them. I have actually not removed this file. Marking PR as draft until then...